File: python/입문/합성수_찾기.py
# 합성수 찾기 문제

def solution(n):
    answer = len([n1 for n1 in range(1,n+1) if len([n2 for n2 in range(1,n1+1) if n1 % n2 == 0]) >= 3])
    return answer

# all()

File: java/입문/합성수_찾기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        for(int i = 1; i <= n; i++) {
            // 약수 개수 구하기
            int count = 0;
            for(int j = 1; j <= i; j++) {
                if (i % j == 0) {
                    count++;
                }
            }
            // 합성수 개수 구하기
            if(count >= 3) {
                answer++;
            }
        }
        return answer;
    }
}